Src/WitsmlExplorer.Frontend/services/jobService.tsx

```diff
@@ -35,14 +35,23 @@ export default class JobService {
     }
   }
 
-  public static async getJobInfos(abortSignal?: AbortSignal): Promise<JobInfo[]> {
+  public static async getUsersJobInfos(abortSignal?: AbortSignal): Promise<JobInfo[]> {
     const response = await ApiClient.get(`/api/jobs/jobinfos`, abortSignal);
     if (response.ok) {
       return response.json();
     } else {
       return [];
     }
   }
+
+  public static async getAllJobInfos(abortSignal?: AbortSignal): Promise<JobInfo[]> {
+    const response = await ApiClient.get(`/api/jobs/jobinfosallusers`, abortSignal);
```

Comment:
A suggestion here is to align the names to match for endpoint with method names. If you think it is a good idea, something like this could be used. E.g. 
`getAllJobInfos`
`/api/jobs/alljobinfos`